src/Symfony/Component/Form/Tests/CompoundFormTest.php

```diff
@@ -902,8 +902,8 @@ public function testClearErrorsShallow()
 
         $this->form->clearErrors(false);
 
-        $this->assertCount(0, $this->form->getErrors(false));
```

Comment:
Tests for the deprecated APIs must be kept (marking them as `@group legacy`). Otherwise, we don't have any guarantee that the BC layer works.